examples/timestamping/backend/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "exonum-timestamping"
-version = "0.0.0"
+version = "1.0.0-rc.1"
```

Comment:
Because we use it as an artefact which has a version as well. There was an error when I've tried to deploy it with version `0.0.0`. By the way I even don't know if it's ok. Maybe it's a bug? What do you think ?

Reply:
Artifact deployment is a good reason to have a "real" version, but not being able to deploy an artifact with version 0.0.0 looks strange and may be worth investigating.